receiver/otlpreceiver/README.md

```diff
@@ -1,12 +1,18 @@
 # OTLP Receiver
 
-| Status                   |                       |
-| ------------------------ | --------------------- |
-| Stability                | traces [stable]       |
-|                          | metrics [stable]      |
-|                          | logs [beta]           |
-| Supported pipeline types | traces, metrics, logs |
-| Distributions            | [core], [contrib]     |
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [beta]: logs   |
+|               | [stable]: traces, metrics   |
+| Distributions | [core], [contrib] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Fotlp%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Areceiver%2Fotlp) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Fotlp%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Areceiver%2Fotlp) |
```

Comment:
This link contains "contrib" in it. Probably we should configure the github org and repo?

Reply:
Yeah, let me fix that

Reply:
Opened an issue to track this work https://github.com/open-telemetry/opentelemetry-collector/issues/9240